File: front-end/src/app/service-models/app/v1/client/client-employee-bank-detail-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ClientEmployeeBankDetailSM extends CoinManagementServiceModelBase<number> {
    bankName!: string;
    branch!: string;
    accountNo!: number;
    ifscCode!: string;
    clientUserId!: number;
    clientCompanyDetailId!: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/ClientEmployeeBankDetailSM.cs
using Siffrum.Web.Payroll.ServiceModels.Base;

namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class ClientEmployeeBankDetailSM : SiffrumPayrollServiceModelBase<int>
    {
        public string BankName { get; set; }

        public string Branch { get; set; }

        public long AccountNo { get; set; }

        public string IfscCode { get; set; }

        public int ClientUserId { get; set; }
        public int ClientCompanyDetailId { get; set; }
    }
}
